test/Keccak256.spec.ts

```diff
@@ -39,6 +50,27 @@ describe('Keccak256 tests', function () {
         await keccakTest.zeroPointerTest()
     });
 
+    it('general functionality test', async () => {
+        // We currently do not have fussing support, so we generate random data using
```

Comment:
Agree that it is non-trivial to integrate Echidna or other Fuzz tooling at the moment. But this test seems to me weird. data is always 32 bytes long. I would rather make a couple of randomly generated tests:
- 0 bytes long 
- x * `BLOCK_SIZE` - 2
- x * `BLOCK_SIZE` - 1
- x * `BLOCK_SIZE`
- x * `BLOCK_SIZE` + 1
- x * `BLOCK_SIZE` + 2

It is to cover all possible variants of branching